src/main/app/Resources/translations/security.fr.json

```diff
@@ -11,5 +11,18 @@
   "newPassword": "L'utilisateur username a modifié son mot de passe.",
   "viewAs": "L'utilisateur username a utilisé la fonctionnalité 'voir en tant que' avec le Role role",
   "validateEmail": "L'utilisateur username a validé son adresse email",
-  "authenticationFailure": "Connexion échoué pour l'utilisateur username : message"
+  "authenticationFailure": "Connexion échoué pour l'utilisateur username : message",
+  "switchUser": "L'utilisateur username a utilisé la fonctionnalité 'voir en tant que' avec l'utilisateur target",
+  "event.security.add_role": "evenement.securite.ajout_role",
```

Comment:
Should be human readable : "Ajout d'un rôle", "Connexion utilisateur" etc.
You will also need the english translations.

Reply:
Change done, thx :)